content/overview.md

```diff
@@ -0,0 +1,62 @@
+threads operate simultaneously to perform fast scientific computations. 
+This is an opportunity, as real-time applications such as low latency triggers can be 
+developed using commodity hardware and software tools. At the same time GPUs are a promising technology with which to run simulation and data processing 
+at lower cost and better power efficiency. To unlock these potential gains an aggressive software R&D programme is needed to modernise the current 
+HEP codes and optimise their performance on the heterogeneous hardware architectures that are expected to be developed in the next decade. <br>
+
+Such developments need to be sustainable, to last for the lifetime of the current experiments, to be open source, to allow for collaboration between different teams, 
+and to follow industry standards, to exploit state-of-the-art training and enable collaboration with industry to access and inform the latest technologies. <br>
+
+Establishing a software collaboration within HEP in the UK will ensure that future investments in HEP experiments will have access to the technology needed 
+to exploit the data to maximise the physics return on investment. This will also create a core of knowledge and training in skills that 
+are in high-demand in UK industry.
+
+
+<h2> Timeline and funding</h2>
+The SWIFT-HEP project covers this crucial 3 year period between April 2021 and March 2024 for the UK to participate and lead the development of the means to execute the next generation experiments. Funding was awarded for an average of 4 FTE software developers/ applied physicists and was awarded at various universities in the UK. 
+ Staff are funded at the 50% level by SWIFT-HEP, with the remaining 50% funded by an experiment grant to conduct work specific to such experimental collaboration. This ensures transfer of knowledge and expertise from the experiments to SWIFT-HEP and vice versa. <br>
+The timeline for the SWIFT-HEP project is summarised in the figure below. 
+SWIFT-HEP kicked off in April 2021, after amm STFC funded pilot project (ECHEP) and some additional funding from the UKRI ExCALIBUR initiative. The funding beyond April 2024 is still under discussion, however the proposal submitted in 2020 made it clear that the project need to continue and expand for the next decade. 
+<center> {{< figure src="/images/swift-timeline.png" alt="swift-hep timeline" width="600px">}} </center>
+
+<h2> Work packages and activities</h2>
+The SWIFT-HEP work is organised in 5 work packages (WP), plus a management WP. A description and status of each WP is given in the dedicated pages. 
+Each of the WPs has two conveners who facilitate communication within the UK community and interaction with external collaborators. 
+(<a href="/organisation/">See also</a>).
+<ul>
+<li> WP0: Management
+<li> WP1: Intelligent data and workflow management
+<li> WP2: Event generators
+<li> WP3: Simulation
+<li> WP4: Reconstruction and trigger
+<li> WP5: Analysis systems
+</ul>
```

Comment:
List should be as simple as

```markdown

- WP0: Management
- WP1: Intelligent data and workflow management
- WP2: Event generators
- WP3: Simulation
- WP4: Reconstruction and trigger
- WP5: Analysis systems

```

If you wanted to be fancy, you could even do a Hugo for-loop ;)